tests/testthat/test-ncbi_snp_api.R

```diff
@@ -1,11 +1,11 @@
-context("ncbi_snp_query_api")
+context("ncbi_snp_query")
 
-test_that("ncbi_snp_query_api for rs1173690113 (merged into rs333)", {
+test_that("ncbi_snp_query for rs1173690113 (merged into rs333)", {
   ## truth: https://www.ncbi.nlm.nih.gov/snp/rs1173690113
   
   skip_on_cran()
   
-  expect_warning(aa <- ncbi_snp_query_api("rs1173690113"), "rs1173690113 has been merged into rs333")
+  expect_warning(aa <- ncbi_snp_query("rs1173690113"), "rs1173690113 has been merged into rs333")
```

Comment:
Hi @sinarueeger,
This is not related to your changes, but it seems that one of the gene symbols has been updated from LOC102724297 to CCR5AS. Do you think it could be possible to include it in this pull request?